nrc_exporter.py

```diff
@@ -490,7 +524,7 @@ def arg_parser():
     ap.add_argument("-v", "--verbose", action="store_true", help="print verbose output")
     ap.add_argument("-t", "--token", help="your nrc token", required=False)
     ap.add_argument(
-        "-i", "--input", help="A directory containing NRC activities in JSON format"
+        "-i", "--input", nargs='+', help="A directory containing NRC activities in JSON format or NRC JSON files"
```

Comment:
Please add more to the help message regarding the fact that people can also specify multiple folders containing the JSON files. Maybe we can change this to: "A directory or directories containing NRC activities in JSON format. You can also provide individual NRC JSON files"

Thoughts?

Reply:
Cool,I will change it right now.